### DISCUSSION TITLE
spoiler feature demo doesn't work in the docs

### DISCUSSION THREAD

**Discussion by StefanoCecere** [vuepress-theme-hope/vuepress-theme-hope#4119] [Q&A] [answered]
https://theme-hope.vuejs.press/guide/markdown/stylize/spoiler.html#settings

i don't see it working in the docs.

**Comment by Mister-Hope** [comment, marked as answer]
I just forgot to turn on it,  It is working in plugin md-enhance docs,  And the fix is already made in main before you open this.

**Reply by StefanoCecere** [reply]
got it https://plugin-md-enhance.vuejs.press/guide/stylize/spoiler.html#settings
you know... the docs are a bit too "duplicated" into too many sub sites :)

**Reply by Mister-Hope** [reply]
The plugin is mean to be used with other themes like vuepress-plugin-plume.  So it must has its own docs,  And that plugin is complicated enough so that it must has its own documentation.

But yes, things are duplicated in both plugin docs and theme docs.  Before I started doing this, a lot of users are complaining that they can not found related instructions or related docs.  So this is kind of trade off,  I choose to include everything that my theme users need, by copying them or simply using the include feature I built.
